loader/src/sources/riteaid/api.js

```diff
@@ -121,9 +215,13 @@ function formatStore(provider) {
     state: provider.location.state,
     postal_code: provider.location.zipcode,
     county,
-    booking_phone: provider.contact.booking_phone,
+    booking_phone:
+      provider.contact.booking_phone &&
+      parseUsPhoneNumber(provider.contact.booking_phone),
```

Comment:
Ah, I was trying to match the behavior of `parseUsAddress`, which also does not support non-string input. It also seems a little easier to reason about if you can guarantee the return type will be a string.

But for this use case, it would definitely be simpler. Think I should update `parseUsAddress`, too? (I think the non-support for null/undefined in `parseUsAddress` is currently supporting some edge cases in NJVSS and Albertsons, so it may require some more changes.)

Reply:
One approach that can solve for that is to allow the caller to provide a default fallback value, or else the function will throw an error given bad input (including wrong data type).

That said, `parseUsAddress` is complicated enough and used elsewhere enough that I don't know if it's worth messing with. I do appreciate that the two `parseUsXYZ` functions are consistent in whether or not you need to look before you leap.

Reply:
In that case, I think I'll keep it as-is for now, and maybe find ways to handle `null` for both of them later.